docs/joss_paper/paper.md

```diff
@@ -0,0 +1,120 @@
+    affiliation: "1, 2"
+affiliations:
+ - name: Wellcome/EPSRC Centre for Surgical and Interventional Sciences, University College London
+   index: 1
+ - name: Centre for Medical Image Computing, University College London
+   index: 2
+ - name: InstaDeep
+   index: 3
+ - name: Department of Surgical & Interventional Engineering, King’s College London
+   index: 4
+date: 12 August 2020
+bibliography: paper.bib
+---
+
+# Summary
+Image fusion is a fundamental task in medical image analysis and computer assisted intervetnion. Medical image registration, a class of algorithms to align different images together, has in recent years turned the research attention towards deep learning. Indeed, the representation ability to learn from population data with deep neural networks opened new possibilities for accelerated registration excuation during inference, improved registration accuracy and more robust generalisation that is less dependent on hand-engineered image features and similarity measures, for feature-based- and intensity-based image registration algorithms, respectively.
+
+`DeepReg` is a Python package that implements a class of image registration algorithms based on deep neural networks. Both unsupervised algorithms [@de2019deep] and weakly-supervised algortihms [@hu2018label] have been implemented. In addition, combining unsupervised losses, such as those derived from image similarity measures, and weak supervision, such as overlap between corresponding anatomical and pathological regions, is at the core of this framework. In this paper, we outline the algorithms and several representative clinical applications, avalibale as 'demos' at the time of writing. An summary of data input and output is also provided to describe the implemented dataset loaders and training data sampling methods. Collectively, we demonstrate the versatility of the package, by providing a wide range of clinical applications as example demonstrations in the `demos` folder.
+
+`DeepReg` is primarily a command line tool that provides basic training and prediction functionalities, with options and configurations being specified though the command line arguments and/or configuration files in .yaml format.
+
+# Algorithms
+## Unsupervised learning
+Unsupervised learning was first developed independently from a number of research groups, among which, [@de2019deep] demonstrated the applicability in real clinical applications while VoxelMorph [@balakrishnan2019voxelmorph] has contributed to a popular open-source repository for several algorithms tested using brain MR images. Image dissimilarity is measured between the fixed and warped moving images, which has been adapted from the classical image registration methods. Fig.\autoref{fig:unsupervised} shows a schematic illustration of the network training. Image dissimilarity measures include sum-of-square difference in intensity (SSD), normalised cross-correlation (NCC), mutual information (MI) and their variants.
```

Comment:
Image dissimilarity measures include, here it should be measurements?

Reply:
this is a sort of a term "dissimilarity measures"